src/components/Main/Sort.js

```diff
@@ -1,13 +1,58 @@
 import React from 'react';
 import '../../styles/sort.scss';
 
-class Sort extends React.PureComponent {
+class Sort extends React.Component {
+    constructor(props) {
+        super(props);
+        this.onSort = this.onSort.bind(this);
+
+        //Default sorting
+        this.props.sort(this.sortByDate(this.props.movies));
```

Comment:
there might be cases when props aren't ready when constructor is fired. Think about moving this to lifecycle method

Reply:
Fixed, thank you!